src/compositions/FieldsetHeader/FieldsetHeader.tsx

```diff
@@ -60,16 +60,24 @@ const renderType = (
             return (
                 <div
                     className={merge([
-                        "tw-text-black-80 tw-transition-transform",
-                        active && "tw-text-black tw-rotate-180 tw-duration-300",
+                        "tw-transition-transform",
+                        active && "tw-rotate-190 tw-duration-300",
+                        active ? "tw-rotate-180 " : "tw-text-black-80",
+                        disabled && !active && "tw-text-black-40",
```

Comment:
`!active` is now set to two different colors if disabled and there's no color change if the state is `active`